manifests/logs.pp

```diff
@@ -37,7 +37,7 @@
     ifempty       => false,
     missingok     => true,
     sharedscripts => true,
-    postrotate    => '/etc/init.d/nginx reopen_logs',
+    postrotate    => '/etc/init.d/nginx reopen_logs 2>/dev/null || /bin/kill -USR1 `cat /var/run/nginx.pid 2>/dev/null` 2>/dev/null || true',
```

Comment:
I think they want a true condition always.  I am just modeling after the new logrotate:

/var/log/nginx/*log {
    create 0644 nginx nginx
    daily
    rotate 10
    missingok
    notifempty
    compress
    sharedscripts
    postrotate
        /bin/kill -USR1 `cat /run/nginx.pid 2>/dev/null` 2>/dev/null || true
    endscript
}